remotespark/kernels/kernelmagics.py

```diff
@@ -147,6 +147,16 @@ def info(self, line, cell="", local_ns=None):
 
         self.ipython_display.writeln("Session configs:\n\t{}\n".format(conf.get_session_properties(self.language)))
 
+        if self.session_started:
+            app_id = self.spark_controller.get_app_id()
+            driver_log_url = self.spark_controller.get_driver_log_url()
+            spark_ui_url = self.spark_controller.get_spark_ui_url()
+        else:
+            app_id = driver_log_url = spark_ui_url = "Session hasn't started."
+        self.ipython_display.writeln("Driver log:\n\t{}\n".format(driver_log_url))
+        self.ipython_display.writeln("Spark UI:\n\t{}\n".format(spark_ui_url))
+        self.ipython_display.writeln("YARN Application ID:\n\t{}\n".format(app_id))
```

Comment:
I'm unsure about the output here.  The user wants the spark UI link and yarn application id for all sessions, not just the default.

That output should become a part of `LivySession` and included in the `__str__` method for it.